### PR TITLE
Change models into folder Models

### PR DESCRIPTION
**Pull request by pipinamngua**

### CODE REVIEW

**Comment by pipinamngua** [comment]
change models into folder Models

**Comment by pipinamngua** [comment]
ready

File: app/Models/Category.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+use Illuminate\Database\Eloquent\Model;
+
+class Category extends Model
+{
+    protected $fillable = [
+        'name',
+        'parent_id',
+    ];
+
+    public function child()
```
Comment (trangkarol) [inline app/Models/Category.php]: tên function mà `hasMany` là số nhiều e nhé

File: app/Models/New.php [hunk 2]
```diff
@@ -1,13 +1,13 @@
 <?php
 
-namespace App;
+namespace App\Models;
 
 use Illuminate\Database\Eloquent\Model;
 
-class New extends Model
+class new extends Model
```
[the remark this answers is not in the document]
Comment (pipinamngua) [reply app/Models/New.php]: ok chị Hằng xink đẹp

File: app/Models/Category.php (hunk 1 above)
Comment (pipinamngua) [reply app/Models/Category.php]: ok chị Trang xink gái

**Comment by pipinamngua** [comment]
rename news into posts table,
dowload retrine/dbal library

**Comment by pipinamngua** [comment]
ready

**Comment by pipinamngua** [comment]
rename news into posts table,
dowload retrine/dbal library

**Comment by pipinamngua** [comment]
ready

**Pull request merged**